inc/configuration/class-yoast-acf-analysis-configuration.php

```diff
@@ -23,17 +28,13 @@ class Yoast_ACF_Analysis_Configuration {
 	protected $scraper_config = array();
 
 	/**
-	 * @var array Field names to exclude from the analysis
-	 */
-	protected $excluded_fields = array();
-
-	/**
-	 * @param Yoast_ACF_Analysis_String_Store  $blacklist       Blacklist Configuration Object.
+	 * @param Yoast_ACF_Analysis_String_Store $blacklist_type  Blacklist Type Configuration Object.
 	 * @param Yoast_ACF_Analysis_String_Store $field_selectors Field Selectors Configuration Object.
 	 */
-	public function __construct( Yoast_ACF_Analysis_String_Store $blacklist, Yoast_ACF_Analysis_String_Store $field_selectors ) {
-		$this->blacklist       = $blacklist;
+	public function __construct( Yoast_ACF_Analysis_String_Store $blacklist_type, Yoast_ACF_Analysis_String_Store $field_selectors ) {
+		$this->blacklist_type  = $blacklist_type;
 		$this->field_selectors = $field_selectors;
+		$this->blacklist_name  = new Yoast_ACF_Analysis_String_Store();
```

Comment:
The Store for the blacklist should not be instantiated here but just like the other two stores in [`Yoast_ACF_Analysis`](https://github.com/Yoast/yoast-acf-analysis/blob/4f8e3fc6823b25038248d2a3c9bd5f02dd6b6e64/inc/class-yoast-acf-analysis.php#L52-L55) and passed as a parameter to the constructor.

Be aware - a lot of tests will fail after that change because they lack the third parameter sou you'll need to update those tests.